src/Payments.Mvc/Views/Payments/Pay.cshtml

```diff
@@ -1,19 +1,92 @@
 ﻿@model Payments.Mvc.Models.PaymentViewModels.InvoicePaymentViewModel
 
 @{
-    ViewBag.Title = "title";
-    Layout = "_Layout";
+    ViewBag.Title = $"Pay Invoice {Model.Id}";
+    Layout = "_PaymentsLayout";
 }
 
-<h2>title</h2>
-<form action="@ViewBag.CyberSourceUrl" method="post" autocomplete="off" style="margin-right: 3px">
-    @foreach (var pair in Model.PaymentDictionary)
+<div class="container">
+    <h1>Invoice from UCDavis Arboretum</h1>
+    <p>Billed to jpknoll@ucdavis.edu</p>
```

Comment:
Temporary i'm assuming

Reply:
Yeah, just trying to get a baseline for Cal to work off of.